app/views/roasts/index.html.erb

```diff
@@ -0,0 +1,70 @@
+<%= render partial: "shared/partials/header-global", locals: {title: "The right roast is out there.", text: "But with so many different types and brands, it can be hard to find. Let us suggest the perfect roast. Slide the thingy."} %>
```

Comment:
If this is on every page except the Landing page, this should be moved to application.html.erb